document/CoreFormat.mdk

```diff
@@ -361,12 +361,70 @@ A `result` object describes a single result produced by a static analysis tool.
 ### `ruleId` property { #result-ruleId }
 
 A `result` object shall contain a property named `ruleId` whose value is a string containing
-a stable, opaque identifier for the rule that was evaluated to produce the result.
+the stable, opaque identifier for the rule that was evaluated to produce the result.
 
 ~ IsoExample
 `"CA2101"`
 ~
 
+### `kind` property { #result-kind }
+
+A `result` object may contain a property named `kind` whose value is one of a fixed set of strings
+that specify the category of the result.
+
+If present, the `kind` property shall have one of the following values, with the specified meanings:
+
+* `pass`: The rule specified by the `ruleId` property (&sect;[#result-ruleId]) was evaluated, and no problem was found.
+
+* `warning`: The rule specified by the `ruleId` property was evaluated, and a problem was found.
+
+* `error`: The rule specified by the `ruleId` property was evaluated, and a serious problem was found.
+
+* `pending`: The rule specified by the `ruleId` property was evaluated, but whether the result of the evaluation
```

Comment:
Note that pending is intended to help decouple policy from results production in cases where it isn't desirable to tightly couple policy with the tool. Canonical example: a tool that reports an observation around the compiler toolchain used to build a binary coupled with a separate policy that requires a min-bar tool chain to 'pass'